README.md

```
@@ -158,6 +158,7 @@ Name | Description
 
 Name | Type | Description
 --- | --- | ---
+[redhatci.ocp.find_available_port]() | Module | A module to find an available port in a given range
```

markdownlint-cli2 (0.22.1)

[warning] No empty links

(MD042, no-empty-links)